src/stages/stage5/nonspells/boss_nonspell_2.c

```diff
@@ -10,55 +10,58 @@
 
 #include "nonspells.h"
 
-MODERNIZE_THIS_FILE_AND_REMOVE_ME
-
 static cmplx bolts2_laser(Laser *l, float t) {
 	if(t == EVENT_BIRTH) {
+		l->shader = res_shader_optional("lasers/iku_lightning");
```

Comment:
Lasers shaders are obsolete now right? This causes
```
  W: _get_resource: shader program 'lasers/iku_lightning' was not preloaded
102948    D: vfs_set_error: Node 'res/shader/lasers/iku_lightning.prog' does not exist
102948    E: load_shader_program_stage1: VFS error: Node 'res/shader/lasers/iku_lightning.prog' does not exist
```

Reply:
Fixed!